test/elastic-query-builder.test.js

```diff
@@ -138,4 +138,149 @@ describe('ElasticQueryBuilder', () => {
         .include({ 'bool.filter[0].term.buildingLocationIds': 'ma' })
     })
   })
+
+  describe('Advanced Search query params', () => {
```

Comment:
I think it would be a good idea to test a combination of filter and regular query param values

Reply:
Added a test